applications/nrf_desktop/src/hw_interface/Kconfig.buttons_sim

```diff
@@ -0,0 +1,38 @@
+#
+# Copyright (c) 2019 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+menu "Buttons simulation configuration"
+
+config DESKTOP_BUTTONS_SIM_ENABLE
+	bool "Enable simulated button presses generator"
+	depends on CAF_BUTTONS
```

Comment:
`depends on !CAF_BUTTONS`

Reply:
also `depends on CAF_BUTTON_EVENTS`

Reply:
or `select CAF_BUTTON_EVENTS`

Reply:
I think `depends on CAF_BUTTON_EVENTS` is the most secure way of doing so. In theory nothing prevents from configuring buttons and simulated buttons at once so it might be either simulated or actual buttons.

Reply:
In fact this configuration (real + simulated buttons) is used: `configuration/nrf52840dk_nrf52840/app_ZDebug_keyboard.conf`. Buttons sim is `nrf_desktop` specific and actually depends on `CAF_BUTTONS`, button press is used to start/stop simulation. I don't think `depends on CAF_BUTTONS` should be removed from here.